amr-wind/core/FieldFillPatchOps.H

```diff
@@ -397,6 +403,62 @@ public:
         }
     }
 
+    void set_inflow_sibling_fields(
+        int lev,
+        amrex::Real time,
+        amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM> mfabs,
+        const amrex::IntVect& nghost) override
+    {
+        const int ng = nghost[0];
```

Comment:
if we are passing an intvect, it makes more sense to move this `ng` and `domain` into the oit loop and get the nghost[dir] there.